content/en/docs/reference/node/kubelet-files.md

```diff
@@ -0,0 +1,114 @@
+---
+content_type: "reference"
+title: Files written and read by the kubelet
+weight: 40
```

Comment:
`content/en/docs/reference/node/node-labels.md` uses 40 and we like the sort to be stable even if the title changes, so:
```suggestion
weight: 42
```